### PR TITLE
Web client and travis

### PR DESCRIPTION
**Pull request by odeimaiz**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,8 +1,9 @@
...
-    },
+{
+    "python.linting.pylintEnabled": true,
+    "python.linting.enabled": true,
+    "files.associations": {
+        ".*rc": "ini",
+        ".env*": "ini"
+    },
+    "files.eol": "\n",
 }
```
Comment (oetiker) [inline .vscode/settings.json]: hihi  that file itself does not seem to have profited from the setting yet :)

Comment (odeimaiz) [reply .vscode/settings.json]: Good point!

**Comment by odeimaiz** [comment]
@oetiker Any comments on the directory structure?

File: .vscode/settings.json (hunk 1 above)
Comment (oetiker) [reply .vscode/settings.json]:
```
    "files.trimTrailingWhitespace": true,
    "files.insertFinalNewline": true,
    "files.trimFinalNewlines": true,
```-

**Pull request merged**